src/theme.tsx

```diff
@@ -26,12 +26,9 @@ const theme = createMuiTheme({
         secondary: {
             main: "#90756d",
         },
-        pronounced: {
-            main: "white",
-        },
     },
     typography: {
-        fontFamily: ['"Secular One"', "sans-serif"],
```

Comment:
I think the fallback is important for older browsers (not sure I remember correctly)

Reply:
It see  that the type not support that
Maybe there anthor way that was intended by mui